models/tmp/stg_shopify__discount_code_tmp.sql

```diff
@@ -17,21 +8,8 @@
         default_database=target.database,
         default_schema='shopify',
         default_variable='discount_code_source',
+        column_macro=get_discount_code_columns(),
```

Comment:
should i add the `column_macro` arg to every single _tmp model? i realized that because the non-tmp staging model will call the column_macro and `fill_staging_column` again, it will have all the expected columns even if the _tmp table only has the `_dbt_source_relation` field

so basically, the `column_macro` field permits the _tmp model to have all the empty fields. without it, only the non-tmp model will have all of them (and the _tmp will just have _dbt_source_relation)

Reply:
I actually think omitting the `column_macro` entirely makes the most sense to help streamline and standardize our code. Additionally, the `column_macro` is working to ensure this functions correct when the source doesn't exist anywhere else, but since you already got it to work if the argument does not exist, I saw we simplify things and stick with the no `column_macro` version.

Reply:
Same comments for the other models.

Reply:
removed